account_statement_base_import/views/account_move_view.xml

```diff
@@ -0,0 +1,67 @@
+<odoo>
+    <record id="view_move_importer_form" model="ir.ui.view">
+        <field name="name">account.move.view</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <field name="journal_id" position="after">
+                <field name="used_for_completion" invisible="1"/>
+            </field>
+            <button name="button_cancel" position="after">
+                <button name="button_auto_completion"
+                        string="Auto Completion"
+                        type="object"
+                        class="oe_highlight"
+                        groups="account.group_account_invoice"
+                        attrs="{'invisible': ['|', ('used_for_completion','=',False), ('state','not in', ['draft'])]}"/>
+            </button>
+            <xpath expr="/form/sheet/notebook/page/field[@name='line_ids']/tree/field[@name='credit']" position="after">
```

Comment:
In such case I prefer to use more relative paths.
Here you could have:
`//field[@name='line_ids']/tree/field[@name='credit']`
Thus if someone move the field outside notebook it will still work.